WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml

```diff
@@ -288,7 +288,7 @@
         <designguidance:ColorPageExample
             Title="Focus Stroke"
             Background="{ThemeResource SolidBackgroundFillColorQuarternaryBrush}"
-            Description="Used for divider and graphic lines. Theme inverse; dark in light theme and light in dark theme..">
+            Description="Used for divider and graphic lines. Theme inverse; dark in light theme and light in dark theme.">
```

Comment:
```suggestion
            Description="Used for divider and graphic lines.">
```

If we remove it for `Divider Stroke` should we remove it here as well? Or leave them in for both?

Reply:
Nope. Here, the claim is **invalid**: the color is lighter in the light theme. The other one is correct.

Reply:
Ah yes, you're right! Nvm